esmvalcore/preprocessor/_regrid.py

```diff
@@ -306,6 +307,48 @@ def _attempt_irregular_regridding(cube, scheme):
     return False
 
 
+def extract_location(cube, location, scheme):
+    """Extract a point using a location name, with interpolation.
+
+    Extracts a single location point from a cube, according
+    to the interpolation scheme `scheme`.
+
+    The function just retrieves the coordinates of the location and then calls
+    the `extract_point` preprocessor.
+
+    It can be use to locate cities and villages, but also mountains or other
+    geographical locations.
+
+    Parameters
+    ----------
+    cube : cube
+        The source cube to extract a point from.
+
+    location : str
+        The reference location.
+
+    scheme : str
+        The interpolation scheme. 'linear' or 'nearest'. No default.
+
+    Returns
+    -------
+    Returns a cube with the extracted point, and with adjusted
+    latitude and longitude coordinates.
+
+    Raises
+    ------
+    ValueError:
+        If given location can not be found by the geolocator.
+    """
+    geolocator = Nominatim(user_agent='esmvalcore')
+    location = geolocator.geocode(location)
+    if not location:
+        raise ValueError(f'Requested location {location} can not be found.')
+    logger.info("Extracting data for %s (%s ºN, %s ºW)", location,
```

Comment:
```suggestion
    logger.debug("Extracting data for %s (%s ºN, %s ºW)", location,
```

Usually we don't show messages of individual preprocessors in the info log.